routes/api.js

```diff
@@ -1,44 +1,34 @@
 'use strict';
 
-var apiConfig = require('../server/config').api;
-var COCKTAIL_PATH = apiConfig.base + apiConfig.cocktailPath;
-var dbOperations = require('../server/db-operations');
+const apiConfig = require('../server/config').api;
+const COCKTAIL_PATH = `${apiConfig.base}${apiConfig.cocktailPath}`;
```

Comment:
Here's an ES6 template string, which allows embedded expressions. All you need is 1.) backticks instead of quotation marks and 2.) `${}` around the embedded content.